assets/javascripts/discourse/components/emoji-value-list.js.es6

```diff
@@ -0,0 +1,223 @@
+import Component from "@ember/component";
+import I18n from "I18n";
+import { isEmpty } from "@ember/utils";
+import { on } from "discourse-common/utils/decorators";
+import { emojiUrlFor } from "discourse/lib/text";
+import { set, action } from "@ember/object";
+import { later } from "@ember/runloop";
+
+export default Component.extend({
+  classNameBindings: [":value-list"],
+  inputDelimiter: null,
+  collection: [],
```

Comment:
you can define an array here, only basic types like boolean, null, integers, ...

do collection: null and set it to an array in init